dev/NavigationView/NavigationViewItem.cpp

```diff
@@ -420,12 +420,13 @@ void NavigationViewItem::OnLostFocus(winrt::RoutedEventArgs const& e)
     }
 }
 
-void NavigationViewItem::UpdateIsExpanded(winrt::TreeViewNode node)
+void NavigationViewItem::UpdateIsExpandedForNode(winrt::TreeViewNode node)
```

Comment:
Could pass this variable in a gsl::strict_not_null<winrt::TreeViewNode> to build the not null requirement into the method signature.